HACKING.md

```diff
@@ -106,6 +106,47 @@ otherwise an official image can be run with
 
 FAF should be available at ```http://localhost:8080/faf/```
 
+### Reporting into FAF
+1. Set up your server URL for libreport
+
+    Insert `URL = http://localhost:8080/faf` into `/etc/libreport/plugins/ureport.conf`
+2. Create crash
+
+    Make sure you have ABRT running and that you have autoreporting enabled (`abrt-auto-reporting enabled`)
+    Install package `will-crash` and then execute `will_abort`
+
+This should be enough, so in a few minutes (reports are processed every few minutes) you should see
+spike in your FAF. However there are a few places where it can go wrong, therefore following
+troubleshooting:
+
+
+
+1. Was the problem caught by ABRT on your local machine?
+
+    Check output of `abrt-cli ls`. If you dont see any problems there, probably ABRT is not running on your machine. Try reading https://abrt.readthedocs.io/en/latest/ .
+
+2. Was the problem sent to FAF?
+
+    Check content of `/var/spool/faf/reports`, you should see there some file which name consists of numbers and letters. (If running containers, this need to be check in container. You can jump into container bash by simply running `make sh`) For example, you can execute 'find /var/spool/faf/reports/' and output should look something like this
+
+        /var/spool/faf/reports/
+        /var/spool/faf/reports/saved
+        /var/spool/faf/reports/saved/6ab72295-cc09-4e15-af52-e818f6920314
+        /var/spool/faf/reports/archive
+        /var/spool/faf/reports/incoming
+        /var/spool/faf/reports/deferred
+
+    If you have no such file there, it means that the report was not sent automatically or was sent to a wrong server.
```

Comment:
- s/If you have no such file there/If there is no such file/
- s/or was sent/or it was sent/